components/fd/dmesg_matcher.go

```diff
@@ -11,7 +11,9 @@ const (
 	//
 	// ref.
 	// https://docs.kernel.org/admin-guide/sysctl/fs.html#file-max-file-nr
-	RegexVFSFileMaxLimitReached = `VFS: file-max limit \d+ reached`
+	EventVFSFileMaxLimitReached   = "error_vfs_file_max_limit_reached"
+	RegexVFSFileMaxLimitReached   = `VFS: file-max limit \d+ reached`
```

Comment:
nit, if it's possible, we should not export these constant values and the functions below.

Reply:
done